doc/source/conf.py

```diff
@@ -163,7 +163,7 @@ def prepare_jinja_env(jinja_env) -> None:
 suppress_warnings = ["autoapi.python_import_resolution"]
 autoapi_python_use_implicit_namespaces = True
 autoapi_keep_files = True
-autoapi_render_in_single_page = ["class", "enum", "exception", "function"]
```

Comment:
FMI, does it mean we can't have items like functions in separate pages?

Reply:
yes, we can add this function as own page. i have added it in https://github.com/ansys/scade-apitools/pull/61/commits/445cd962f27d62e516cbd1c64fa2d3d74fa1477e. please confirm the rendered doc is as expected. After that we can merge this PR.

Reply:
I don't see any change, we can stick to "class" if this is the default.

Reply:
functions will show functions in the toc tree as the image below ![image](https://github.com/ansys/scade-apitools/assets/104772255/5e7b1a47-5cb7-4a32-80f9-4bab7b47153a)
This is for class
![image](https://github.com/ansys/scade-apitools/assets/104772255/e1ec8a95-7101-437b-826f-15f532c65602)

Reply:
I understand and I can reproduce, this is much better! It is important to list the functions of the module in the navigation pane.